jekyll/_data/sidenav.yml

```diff
@@ -970,6 +970,10 @@ en:
             link: style/using-tables
           - name: Using code samples
             link: style/using-code-samples
+      - name: Content templates
+        children:
+          - name: Overview page template
```

Comment:
Should we consider indicating "conceptual" here? We mention it in the intro of the template itself which helps, and if it makes sense to also add it in the sidenav that may also be helpful in eventually helping people pick from the various template options

Reply:
Yep good call, thanks!